app/services/api/v3/participant_declarations_query.rb

```diff
@@ -81,7 +83,7 @@ def ecf_previous_declarations_scope
           )
           .where.not(participant_profile: { induction_records: { induction_programmes: { partnership_id: nil } } }) # Exclude any nil partnerships (eg: cip)
           .where(participant_profile: { induction_records: { induction_programme: { partnerships: { lead_provider_id: lead_provider&.id } } } })
-          .where(participant_profile: { induction_records: { induction_status: "active" } }) # only want induction records that are the winning latest ones
+          .where(participant_profile: { induction_records: { induction_status: RELEVANT_INDUCTION_STATUS } }) # only want induction records that are the winning latest ones
```

Comment:
can we also add this to v1/v2 please in `app/services/api/participant_declarations/index.rb`?